app/components/sidebar/SideBarDropdownItem.tsx

```
@@ -12,18 +12,7 @@ interface SideBarDropdownItemProps extends DropdownProps {
 
 const SideBarDropdownItem: FunctionComponent<SideBarDropdownItemProps> = ({ icon, label, style, onOpen, children }) => {
     return (
-        <Dropdown
-            trigger={
-                <SideBarItem>
-                    {icon}
-                    {label}
-                </SideBarItem>
-            }
-            pointing="left"
-            icon={null}
-            fluid
-            onOpen={onOpen}
-        >
+        <Dropdown trigger={<SideBarItem icon={icon} label={label} />} pointing="left" icon={null} fluid onOpen={onOpen}>
```

As I can see from the `DropdownProps`, the icon typing looks like this:
```
  /** Shorthand for Icon. */
  icon?: any
```

So we should be able to not explicitly declare `icon` as `null`

---

When `icon` prop is not specified then default icon is used, passing `null` hides the icon

---

Okay, thanks for the explanation!